ansible_ai_connect/ai/api/model_pipelines/http/pipelines.py

```diff
@@ -96,7 +96,9 @@ def invoke(self, params: CompletionsParameters) -> CompletionsResponse:
                 headers=self.headers,
                 json=model_input,
                 timeout=self.task_gen_timeout(task_count),
-                verify=self.config.verify_ssl,
+                verify=(
+                    self.config.ca_cert_file if self.config.ca_cert_file else self.config.verify_ssl
```

Comment:
I support a more general solution where we can put the trusted ca in a system location